leapp/dialogs/__init__.py

```diff
@@ -19,3 +19,12 @@ class RawMessageDialog(Dialog):
 
     def __init__(self, message):
         super(RawMessageDialog, self).__init__(scope=None, reason=message)
+
+    def request_answers(self, store, renderer):
```

Comment:
This function has a same name, signature and a docstring as one in "leapp/dialogs/dialog.py". Would not it be better to differentiate it? Or in case this is somehow on purpose to mention why it is so?

Reply:
Ahh, ok this is part of functionality...